pkg/commands/fmt.go

```diff
@@ -70,6 +71,7 @@ func newFmtCommand(logger logutils.Log, info BuildInfo) *fmtCommand {
 	setupFormattersFlagSet(c.viper, fs)
 
 	fs.BoolVarP(&c.opts.diff, "diff", "d", false, color.GreenString("Display diffs instead of rewriting files"))
+	fs.BoolVar(&c.opts.diffColored, "diff-colored", false, color.GreenString("Display diffs instead of rewriting files (with colors)"))
```

Comment:
What about having just --color ?

Using --diff --color would be more classic

Reply:
I think about it when I created the flag, but I prefer one flag because it's easier to use.

Reply:
Also because `--color` can only be used with `--diff` :wink: